Networking15.02Escaping/Networking15.02Escaping/Extension/Extension to UIViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Extension to UIViewController.swift
+//  Networking15.02Escaping
+//
+//  Created by Tania Harbarchuk on 23.02.2023.
+//
+
+import Foundation
+import UIKit
+
+extension ViewController: UITableViewDelegate, UITableViewDataSource {
```

Comment:
ViewController is not base class. "Extension" folder **should** contain only extensions to base classes